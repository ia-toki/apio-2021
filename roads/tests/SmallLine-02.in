c10234cabdfd6eae47773fb6f163e0350fc797e1
191
0 1 729078100
1 2 767557971
2 3 180298865
3 4 114953486
4 5 191065394
5 6 940938196
6 7 610880461
7 8 328050996
8 9 183108344
9 10 726774422
10 11 212906788
11 12 321599678
12 13 408399561
13 14 306515309
14 15 992339932
15 16 500835110
16 17 744715358
17 18 396846862
18 19 498201102
19 20 460500205
20 21 938603839
21 22 454924113
22 23 9500868
23 24 76686289
24 25 381274375
25 26 782914927
26 27 614419513
27 28 137381655
28 29 446936995
29 30 929389124
30 31 676979740
31 32 331376578
32 33 367375251
33 34 462998670
34 35 16583116
35 36 124072153
36 37 236664360
37 38 662233899
38 39 777931420
39 40 58552741
40 41 914442692
41 42 253439839
42 43 222649872
43 44 58233327
44 45 541142556
45 46 452164875
46 47 60879902
47 48 770797126
48 49 351633017
49 50 282772214
50 51 766405998
51 52 207655991
52 53 186933234
53 54 826170632
54 55 731834266
55 56 68173458
56 57 678328754
57 58 962496649
58 59 941781870
59 60 140615083
60 61 993534776
61 62 164898918
62 63 455455016
63 64 335261078
64 65 240843410
65 66 760908140
66 67 281394360
67 68 665169514
68 69 936044192
69 70 92995722
70 71 903760385
71 72 793963135
72 73 613778775
73 74 20111635
74 75 649195798
75 76 213853871
76 77 305356794
77 78 828194553
78 79 327011965
79 80 190382744
80 81 657128290
81 82 621652755
82 83 815804783
83 84 309706455
84 85 206079032
85 86 640316247
86 87 102623765
87 88 64172878
88 89 457712949
89 90 182396704
90 91 90820556
91 92 126094686
92 93 124879208
93 94 56794984
94 95 355974497
95 96 41886765
96 97 437512815
97 98 159648014
98 99 752727382
99 100 806341345
100 101 49955516
101 102 847713042
102 103 788769307
103 104 83838691
104 105 316593617
105 106 806646483
106 107 137440340
107 108 637639197
108 109 767114884
109 110 35790423
110 111 151313689
111 112 56913211
112 113 772798779
113 114 774961795
114 115 816888881
115 116 21463021
116 117 112292913
117 118 91393374
118 119 926804565
119 120 814803235
120 121 548485039
121 122 707657298
122 123 892541299
123 124 820899151
124 125 791012435
125 126 922951275
126 127 710763752
127 128 666023078
128 129 526717771
129 130 235085973
130 131 125162948
131 132 460715871
132 133 857855130
133 134 787514849
134 135 923829001
135 136 631783410
136 137 981504338
137 138 16742336
138 139 898530796
139 140 549300569
140 141 688154072
141 142 357864006
142 143 780883986
143 144 344500049
144 145 241765923
145 146 52946536
146 147 465957332
147 148 183086746
148 149 807421437
149 150 372576830
150 151 940563579
151 152 45577011
152 153 827581760
153 154 967575873
154 155 254748803
155 156 392014494
156 157 627559961
157 158 343397005
158 159 812275676
159 160 542340641
160 161 68008775
161 162 78015264
162 163 328189476
163 164 258100994
164 165 932267100
165 166 188560921
166 167 830229742
167 168 960378058
168 169 910151675
169 170 115134201
170 171 169355261
171 172 252344195
172 173 457221144
173 174 87372772
174 175 345214857
175 176 494258682
176 177 479355963
177 178 858004961
178 179 961328248
179 180 778864124
180 181 634708924
181 182 524925205
182 183 625679013
183 184 737908892
184 185 674384391
185 186 742541068
186 187 488533789
187 188 579623531
188 189 228909786
189 190 485042928
