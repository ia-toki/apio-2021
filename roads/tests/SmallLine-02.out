064bde9ff69ddc34b3b45c2d26d58873d85290d3
OK
91692922954 37294969453 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
